app/models/rbac.rb

```diff
@@ -248,7 +248,33 @@ def self.find_targets_with_user_group_rbac(klass, scope, rbac_filters, find_opti
     end
   end
 
+  def self.group(user_or_group)
+    case user_or_group
+    when User
+      user_or_group.current_group
+    when MiqGroup
+      user_or_group
+    when NilClass
```

Comment:
Yes definitely. That would give us the place to add the logic when we're ready in the next phase.